File: Server/asp-net-mvc/Vision/Models/Glasses.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Vision.Models
{
    public class Glasses
    {
        public int GlassesId { get; set; }
        public short Group { get; set; }
        public int Number { get; set; }

        public bool OD_Blind { get; set; }
        public float OD_Spherical { get; set; }
        public float OD_Cylindrical { get; set; }
        public byte OD_Axis { get; set; }
        public float OD_Add { get; set; }

        public bool OS_Blind { get; set; }
        public float OS_Spherical { get; set; }
        public float OS_Cylindrical { get; set; }
        public byte OS_Axis { get; set; }
        public float OS_Add { get; set; }

        public bool? Sunglasses { get; set; }
        public char? Size { get; set; }
        public char? Gender { get; set; }

        public DateTime AddedDate { get; set; }
        public DateTime RemovedDate { get; set; }
        public string RemovedReason { get; set; }
        public string RemovedBy { get; set; }
    }
}

File: app/Vision/src/com/andrewma/vision/models/Glasses.java
package com.andrewma.vision.models;

import java.util.Date;

import com.andrewma.vision.database.core.DbDataType;
import com.andrewma.vision.database.core.annotations.Column;
import com.andrewma.vision.database.core.annotations.PrimaryKey;
import com.andrewma.vision.database.core.annotations.Table;

@Table
public class Glasses {
	@PrimaryKey
	public int GlassesId;

	@Column(dataType = DbDataType.INTEGER)
	public int Group;
	@Column(dataType = DbDataType.INTEGER)
	public int Number;

	@Column(dataType = DbDataType.INTEGER)
	public boolean OD_Blind;
	@Column(dataType = DbDataType.REAL)
	public float OD_Spherical;
	@Column(dataType = DbDataType.REAL)
	public float OD_Cylindrical;
	@Column(dataType = DbDataType.INTEGER)
	public int OD_Axis;
	@Column(dataType = DbDataType.REAL)
	public float OD_Add;

	@Column(dataType = DbDataType.INTEGER)
	public boolean OS_Blind;
	@Column(dataType = DbDataType.REAL)
	public float OS_Spherical;
	@Column(dataType = DbDataType.REAL)
	public float OS_Cylindrical;
	@Column(dataType = DbDataType.INTEGER)
	public int OS_Axis;
	@Column(dataType = DbDataType.REAL)
	public float OS_Add;

	@Column(dataType = DbDataType.TEXT, nullable = true)
	public Boolean Sunglasses;
	@Column(dataType = DbDataType.TEXT, nullable = true)
	public char Size;
	@Column(dataType = DbDataType.TEXT, nullable = true)
	public char Gender;

	@Column(dataType = DbDataType.INTEGER)
	public Date AddedDate;
	@Column(dataType = DbDataType.INTEGER)
	public Date RemovedDate;
	@Column(dataType = DbDataType.TEXT, nullable = true)
	public String RemovedReason;
	@Column(dataType = DbDataType.TEXT, nullable = true)
	public String RemovedBy;
}
